torch/utils/cpp_extension.py

```diff
@@ -85,6 +123,9 @@ def _find_cuda_home():
 
                               !! WARNING !!
 '''
+ROCM_HOME = _find_rocm_home()
+MIOPEN_HOME = _join_rocm_home('miopen') if ROCM_HOME else None
+is_hip_extension = True if ((ROCM_HOME is not None) and (torch.version.hip is not None)) else False
```

Comment:
nit: `is_hip_extension = (ROCM_HOME is not None) and (torch.version.hip is not None)` is sufficient. Also, global variables in this file have capital letters so we should probably stick with that style: IS_HIP_EXTENSION

Reply:
Done